app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,25 @@ def index
     @posts = @user.posts
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    post = Post.new(params.required(:post).permit(:title, :text))
```

Comment:
### Optional
- [ ] It would be nice if we create a method for our `params`. Here is a [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) to help us understand why we need to create a method for our params

Reply:
@addod19 thanks for the review. I now that would be possible, from a UX point of view and comparing to other pages, you add comments without going to a new view. I already apply creating a new view when you create a new post. Thanks for the link to improve calling the params.

Reply:
When there is a clash between Microverse's requirements and your point of view try to put forward the requirement. If you are still not convinced after your project has been approved you can. always make changes

Reply:
@fobadara thanks for the suggestion. Here are the link of the [requirements for this project](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md), as you can see it doesn't specify to where the comment form should be. It would be of great help if the reviewers notify this to Microverse so the project requirements are more specific.